pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -441,6 +441,8 @@ spec:
       runAfter:
         - populate-release-notes
         - embargo-check
+        - run-file-updates
```

Comment:
`set-advisory-severity` uses internal requests when there is a CVE in the releaseNotes. So, usually it may just take a second, but I am not sure it is always that fast